Tasks/month.js

```diff
@@ -15,12 +16,10 @@ Months = [
   'dec',
 ];
 
-Month = (s) => {
-  l = Months.length;
-  for (let i = 0; i < l; i++) {
-    if (s.toLowerCase().startsWith(Months[i])) return i + 1;
-  }
-  return -1;
+const month = (s) => {
+  const month = s.toLowerCase().substring(0, 3);
```

Comment:
Hardcoded magic number: `3`

Reply:
Used assignment for magic number: 3
https://github.com/RostislavHundarev/fixed-exams